green/loader.py

```diff
@@ -153,13 +164,15 @@ def discover(self, current_path, file_pattern="test*.py", top_level_dir=None):
 
         return flattenTestSuite(suite) if suite.countTestCases() else None
 
-    def loadTargets(self, targets, file_pattern="test*.py"):
-        # If a string was passed in, put it into a list.
-        if type(targets) != list:
+    def loadTargets(
+        self, targets: Iterable[str] | str, file_pattern: str = "test*.py"
+    ) -> GreenTestSuite | None:
+        # If a string was passed in, put it into a tuple.
+        if isinstance(targets, str):
             targets = [targets]
 
         # Make sure there are no duplicate entries, preserving order
-        target_dict = OrderedDict()
```

Comment:
Ah, I see that [regular python dicts now have ordered dict functionality by default](https://docs.python.org/3/library/collections.html#ordereddict-objects).

Reply:
OrderedDict has more features but if they are not used, a regular dict is good enough.